lib/src/main/kotlin/com/swmansion/starknet/data/types/SimulatedTransaction.kt

```diff
@@ -65,7 +65,7 @@ data class FunctionInvocation(
     val messages: List<OrderedMessageL2ToL1>,
 
     @SerialName("execution_resources")
-    val computationResources: ComputationResources,
+    val computationResources: ExecutionResources,
```

Comment:
In the spec this still points to non-existent `COMPUTATION_RESOURCES`:
https://github.com/starkware-libs/starknet-specs/blob/647caa00c0223e1daab1b2f3acc4e613ba2138aa/api/starknet_trace_api_openrpc.json#L448

Probably something we should clarify

Reply:
There isn't `COMPUTATION_RESOURCES` schema anymore
https://github.com/starkware-libs/starknet-specs/blob/8cf463b79ba1dd876f67c7f637e5ea48beb07b5b/api/starknet_trace_api_openrpc.json#L465

Reply:
The name's also changed to `Execution resources` there. 

I wonder if we should keep try to keep the old name to avoid breaking change, or do it earlier than later, since the spec is expected to introduce some breaking changes anyways?